custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+  export CLOUDSQL_INSTANCE=${CLOUDSQL_INSTANCE}
+  export DB_USER=${DB_USER}
+  export DB_PASS=${DB_PASS}
+
+  # Build options for simple importer
+  local importer_options=""
+  [[ -n "$INPUT_DIR" ]] && importer_options="$importer_options --input_dir=$INPUT_DIR"
+  [[ -n "$CONFIG" ]] && importer_options="$importer_options --config_file=$CONFIG"
+  [[ -n "$MODE" ]] && importer_options="$importer_options --mode=$MODE"
+  [[ -n "$OUTPUT_DIR" ]] && importer_options="$importer_options --output_dir=$OUTPUT_DIR"
+  mkdir -p $OUTPUT_DIR
+  OUTPUT_DIR=$(readlink -f $OUTPUT_DIR)
+
+  # Clear state from old run
+  report_json=$OUTPUT_DIR/process/report.json
+  rm $report_json
+
+  # Run the simple importer
+  local cwd="$PWD"
+  cd "$SIMPLE_DIR"
+  cmd="python -m stats.main $importer_options"
+  echo_log "Running command: $cmd"
+  $cmd >> $LOG 2>&1
+  cmd_status=$?
+  cd "$cwd"
+  if [[ "$cmd_status" != "0" ]]; then
+    echo_fatal "Failed to run simple importer: $cmd. Logs in $LOG"
+  fi
+
+  status=$(grep '"status"' $report_json)
+  echo_log "simple importer: $status"
+}
+
+# Generate tmcf for a csv file in dir
+function get_or_generate_tmcf {
+  set -x
+  local dir="$1"; shift
+
+  tmcf=$(ls $dir/*.tmcf | head -1)
+  [[ -n "$tmcf" ]] && echo "$tmcf" && return
+
+  # Create a new tmcf for columns in csv
+  csv=$(ls $dir/*.csv | head -1)
+  tmcf=$(echo "$csv" | sed -e 's/.csv$/.tmcf/')
+  # TODO: generate tmcf based on csv columns
+  cat <<END_TMCF > $tmcf
+Node: E:Stats->E0
+typeOf: dcs:StatVarObservation
+observationAbout: C:Stats->entity
+observationDate: C:Stats->date
+variableMeasured: C:Stats->variable
+value: C:Stats->value
+END_TMCF
+  echo "$tmcf"
+  set +x
+}
+
+# Run dc-import genmcf to validate generated csv/mcf files.
+function validate_output {
+  #TODO: For customdc validate data in sql triples db and observations db
+  # get output csv if any.
+  local output_csv=$(ls $OUTPUT_DIR/*.csv 2>/dev/null)
+  if [[ -z "$output_csv" ]]; then
+    echo_log "Skipping validate as there are no csv files in $OUTPUT_DIR"
+    return
+  fi
+  setup_dc_import
+  echo_log "Validating output in $OUTPUT_DIR"
+
+  # Get the tmcf file to validate csv stats.
+  tmcf=$(get_or_generate_tmcf "$OUTPUT_DIR")
+
+  # Run dc-import
+  DC_IMPORT_CMD="genmcf"
+  [[ "$MODE" == "customdc" ]] && DC_IMPORT_CMD="lint"
+  cmd="java -jar $DC_IMPORT_JAR $DC_IMPORT_CMD -n 20 -r FULL $OUTPUT_DIR/*.csv $tmcf -o $OUTPUT_DIR/dc_generated"
+  run_cmd $cmd
+  echo_log "Output of validation in $OUTPUT_DIR/dc_generated/report.json"
+}
+
+# Generate embeddings for sentences
+function generate_embeddings {
+  setup_python
+  EMBEDDINGS_TOOL_DIR=${EMBEDDINGS_TOOL_DIR:-"$WEBSITE_DIR/tools/nl/embeddings"}
+  NL_DIR=${NL_DIR:-"$OUTPUT_DIR/nl"}
+  echo_log "Building embeddings for sentences in $NL_DIR"
+  local cwd="$PWD"
+  cd $EMBEDDINGS_TOOL_DIR
+  run_cmd python3 build_custom_dc_embeddings.py --sv_sentences_csv_path=$NL_DIR/sentences.csv --output_dir=$NL_DIR
```

Comment:
The tool is now a module and will need to be run differently. See https://github.com/datacommonsorg/website/pull/4254

Reply:
Changed to `python -m tools.nl.embeddings.build_custom_dc_embeddings --sv_sentences_csv_path=$NL_DIR/sentences.csv --output_dir=$NL_DIR` 
based on tools/nl/embeddings/run_custom.sh

Let me know if there are different command line params.

Reply:
@shifucun updated the params for this script in https://github.com/datacommonsorg/website/pull/4371 (basically `--input_dir` instead of `sv_sentences_csv_path`. Also see: https://github.com/datacommonsorg/website/blob/83be489138c1c91463b5d80ff0bb3d757b3504dc/tools/nl/embeddings/build_custom_dc_embeddings.md

Reply:
Thanks, changed to --input_dir. Will change for build_embeddings.py as a followup